code/modules/supermatter/supermatter.dm

```diff
@@ -351,6 +351,13 @@
 */
 
 /obj/machinery/power/supermatter/attackby(obj/item/weapon/W as obj, mob/living/user as mob)
+
+	if(istype(W,/obj/item/weapon/tape_roll))
+		playsound(src, 'sound/items/tape.ogg',25)
+		user.visible_message("<span class='notice'>\The [user] applies duct tape to the supermatter, repairing it.</span>")
```

Comment:
Shouldn't this be \the [src] instead of the supermatter?

Reply:
yeah, when an admin renames the supermatter to "kill krystal 90000000" the message should reflect that, this pr is broken otherwise 👎

Reply:
Or even just `[src]` (implied the).